ios/PresentationLayer/Recipes/Tests/RecipesTests/BooksViewModelTests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Created by David Kadlček on 14.04.2023
+//  Copyright © 2023 Matee. All rights reserved.
+//
+
+@testable import Recipes
+import Resolver
+import SharedDomain
+import SharedDomainMocks
+import DevstackKmpShared
+import UIToolkit
+import XCTest
+
+@MainActor
+final class BooksViewModelTests: XCTestCase {
+
```

Comment:
pls `// MARK: Dependencies` to keep it consistent with other tests

Reply:
Done